server/pkg/dal/diff.go

```diff
@@ -1,24 +1,29 @@
 package dal
 
 type (
-	modelDiffType string
-
+	modelDiffType     string
+	ModelModification string
 	// ModelDiff defines one identified missmatch between two models
 	ModelDiff struct {
-		Type modelDiffType
+		Type         modelDiffType
+		Modification ModelModification
 		// Original will be nil when a new attribute is being added
 		Original *Attribute
-		// Asserted will be nil wen an existing attribute is being removed
-		Asserted *Attribute
+		// Inserted will be nil wen an existing attribute is being removed
+		Inserted *Attribute
 	}
 
 	ModelDiffSet []*ModelDiff
 )
 
 const (
-	AttributeMissing              modelDiffType = "attributeMissing"
-	AttributeTypeMissmatch        modelDiffType = "typeMissmatch"
-	AttributeSensitivityMissmatch modelDiffType = "sensitivityMissmatch"
+	AttributeMissing             modelDiffType     = "attributeMissing"
+	AttributeTypeMissmatch       modelDiffType     = "typeMissmatch"
+	AttributeSensitivityMismatch modelDiffType     = "sensitivityMismatch"
+	AttributeCodecMismatch       modelDiffType     = "sensitivityMismatch"
```

Comment:
value probably needs to be "codecMismatch"

Reply:
Good catch. Fixed.